### PR TITLE
Tests sorted. Tested and repaired additional methods in PIResponse.

### PR DESCRIPTION
**Pull request by lukasmatusiewicz**
Branches can be merged to master. Tests work is done for now.

### CODE REVIEW

File: .phpunit.result.cache [hunk 1]
```diff
@@ -1 +1 @@
-{"version":1,"defects":[],"times":{"PrivacyIDEATest::testGetAuthToken":0.08,"PrivacyIDEATest::testEnrollToken":0.118,"PrivacyIDEATest::testPollTransaction":0.052,"PrivacyIDEATest::testTriggerChallenge":0.094,"PrivacyIDEATest::testValidateCheck":0.075}}
\ No newline at end of file
+{"version":1,"defects":{"PrivacyIDEATest::testTriggerChallenge":4,"PrivacyIDEATest::testValidateCheck":3},"times":{"PrivacyIDEATest::testGetAuthToken":0.349,"PrivacyIDEATest::testEnrollToken":0.596,"PrivacyIDEATest::testPollTransaction":0.202,"PrivacyIDEATest::testTriggerChallenge":0.462,"PrivacyIDEATest::testValidateCheck":0.491}}
```
Comment (nilsbehlen) [inline .phpunit.result.cache]: this file can be removed from git

Comment (lukasmatusiewicz) [reply .phpunit.result.cache]: ok.

Comment (lukasmatusiewicz) [reply .phpunit.result.cache]: phpunit.result.cache added to .gitignore and commited

**Review by nilsbehlen — APPROVED** [review]

File: .phpunit.result.cache (hunk 1 above)
Comment (nilsbehlen) [reply .phpunit.result.cache]: Also remove it from the PR please

**Review by nilsbehlen — CHANGES_REQUESTED** [review]

**Pull request merged**